test/unit/services/helpers/test-reduction-helper.js

```diff
@@ -1,20 +1,26 @@
+const moment = require('moment')
 const expect = require('chai').expect
 const Reduction = require('../../../../app/services/domain/reduction')
 const reductionHelper = require('../../../../app/services/helpers/reduction-helper')
-const reductionStatusType = require('../../../../app/constants/reduction-status-type')
 
-var currentDate = new Date()
-var activeStartDate = new Date(new Date().setDate(currentDate.getDate() - 30))
-var activeEndDate = new Date(new Date().setDate(currentDate.getDate() + 30))
-var scheduledStartDate = new Date(new Date().setDate(currentDate.getDate() + 30))
-var scheduledEndDate = new Date(new Date().setDate(currentDate.getDate() + 60))
+var activeStartDate = moment().subtract(30, 'days').toDate()
+var activeEndDate = moment().add(30, 'days').toDate()
+var scheduledStartDate = moment().add(30, 'days').toDate()
+var scheduledEndDate = moment().add(60, 'days').toDate()
+var archivedStartDate = moment().subtract(30, 'days').toDate()
+var archivedEndDate = moment().subtract(15, 'days').toDate()
 
-var ACTIVE_REDUCTION = new Reduction(1, 10, activeStartDate, activeEndDate, 'active note', reductionStatusType.ACTIVE)
-var SCHEDULED_REDUCTION = new Reduction(2, 10, scheduledStartDate, scheduledEndDate, 'scheduled note', reductionStatusType.SCHEDULED)
-var ARCHIVE_REDUCTION = new Reduction(3, 10, new Date(), new Date(), 'archive note', reductionStatusType.ARCHIVED)
-var DELETED_REDUCTION = new Reduction(4, 10, new Date(), new Date(), 'deleted note', reductionStatusType.DELETED)
+var ACTIVE_REDUCTION = new Reduction('1', '11',
+  [activeStartDate.getDate(), activeStartDate.getMonth() + 1, activeStartDate.getFullYear()],
+  [activeEndDate.getDate(), activeEndDate.getMonth() + 1, activeEndDate.getFullYear()], 'active note')
+var SCHEDULED_REDUCTION = new Reduction('2', '12',
+  [scheduledStartDate.getDate(), scheduledStartDate.getMonth() + 1, scheduledStartDate.getFullYear()],
+  [scheduledEndDate.getDate(), scheduledEndDate.getMonth() + 1, scheduledEndDate.getFullYear()], 'scheduled note')
+var ARCHIVE_REDUCTION = new Reduction('3', '13',
+  [archivedStartDate.getDate(), archivedStartDate.getMonth() + 1, archivedStartDate.getFullYear()],
+  [archivedEndDate.getDate(), archivedEndDate.getMonth() + 1, archivedEndDate.getFullYear()], 'archive note')
 
-var REDUCTIONS = [ACTIVE_REDUCTION, SCHEDULED_REDUCTION, ARCHIVE_REDUCTION, DELETED_REDUCTION]
```

Comment:
Any reason `DELETED` was removed? Think it was included to illustrate that a deleted reduction will not get added to any array by the helper